### PR TITLE
Add capability check around avatar handling

### PR DESCRIPTION
**Pull request by nickvergessen**
### ☑️ Resolves

* E.g. Talk 17 Desktop client connecting to a Talk 16 server

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
Bildschirmfoto vom 2023-05-08 17-00-40 | Bildschirmfoto vom 2023-05-08 16-57-49
Bildschirmfoto vom 2023-05-08 17-00-49 | Bildschirmfoto vom 2023-05-08 16-57-56
Bildschirmfoto vom 2023-05-08 17-00-58 | Bildschirmfoto vom 2023-05-08 16-58-05

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not required
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

**Comment by Antreesy** [comment]
Move ConversationAvatarEditor under the description, as it has less priority, and is aligned now with `Create new conversation` template

🏚️ Before | 🏡 After
---|---
image | image

File: src/components/ConversationIcon.vue [hunk 1]
```diff
@@ -59,12 +59,15 @@
...
 import VideoIcon from 'vue-material-design-icons/Video.vue'
 
+import { getCapabilities } from '@nextcloud/capabilities'
 import { generateOcsUrl } from '@nextcloud/router'
 
 import NcAvatar from '@nextcloud/vue/dist/Components/NcAvatar.js'
 
 import { CONVERSATION } from '../constants.js'
 
+const supportsAvatar = getCapabilities()?.spreed?.features?.includes('avatar')
```
Comment (Antreesy) [inline src/components/ConversationIcon.vue:69]: `spreed.features` is Array of strings, so we cannot look for available features by its name as a key

**Review by Antreesy — APPROVED** [review]
🦭

File: src/components/ConversationIcon.vue (hunk 1 above)
Comment (nickvergessen) [reply src/components/ConversationIcon.vue:69]: Ops right

**Pull request merged**